app/main.py

```diff
@@ -2,0 +1,61 @@
+class Animal:
+    animals_list = []
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.animals_list.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(self,
+                 name: str,
+                 appetite: int = 3,
+                 is_hungry: bool = True
+                 ) -> None:
+        super().__init__(name, is_hungry)
+        self.appetite = appetite
+
+    def catch_mouse(self) -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self,
+                 name: str,
+                 appetite: int = 7,
+                 is_hungry: bool = True
+                 ) -> None:
+        super().__init__(name, is_hungry)
+        self.appetite = appetite
+
+    def bring_slippers(self) -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(list_of_animals: Animal.animals_list) -> int:
```

Comment:
The parameter `list_of_animals` should be of type `List[Animal]` instead of `Animal.animals_list`. You can import `List` from `typing` to specify the type hint correctly.